crowd_anki/export/anki_exporter.py

```diff
@@ -67,3 +71,81 @@ def _copy_media(self, deck, deck_directory):
                             str(media_directory.resolve()))
             except IOError as ioerror:
                 print("Failed to copy a file {}. Full error: {}".format(file_src, ioerror))
+
+    def sort_notes(self, notes):
+        deck_sort_config = self.config.get(EXPORT_DECK_SORT, False)
+
+        if not deck_sort_config or "method" not in deck_sort_config:
+            return notes
+
+        sort_method = deck_sort_config["method"]
+        is_reversed = deck_sort_config["reversed"] if "reversed" in deck_sort_config else False
+
+        def _formatted_sort_method(method):
+            return re.sub("[-_\s+]", "", method).lower()
+
+        # Make sure formatting of sort_method is correct, and format each entry
+        if isinstance(sort_method, str):
+            sort_method = [_formatted_sort_method(sort_method)]
+        elif isinstance(sort_method, list):
+            if len(sort_method) == 1:
+                sort_method = [_formatted_sort_method(sort_method[0])]
+            elif len(sort_method) > 1:
+                sort_method = [_formatted_sort_method(sm) for sm in sort_method]                
+            else:
+                return notes
+        else:
+            return notes
+
+        # Create mapping of sortable_values variable, to sort_method keys, to sortable_values lambda
+        sorting_definitions = [
+            (0, ("guid", "guids"), lambda i: i.anki_object.guid),
```

Comment:
I really feel like a separate config validator would better solve the problem you're trying to solve here. Having it inline here adds quite a bit of complexity

Reply:
True true, I'll get on that.